src/components/Menu.jsx

```diff
@@ -30,27 +31,24 @@ export class Menu extends Component {
           </li>
         </React.Fragment>);
     } else {
+      const { user} = currentUser;
+      const { userType} = user;
       links = (
         <React.Fragment>
-          <li className="nav-item offset-md-5">
-            <NavLink to="/notify" ><img src={notify} alt="" /></NavLink>
+          <li className="nav-item">
+            <NavLink to="/notify" ><i className="fa fa-bell m-3"> </i></NavLink>
           </li>
-          <li className="nav-item offset-md-5">
-            <NavLink to="/profile"
-              id="nav-link">{auth.user.username}</NavLink>
-          </li>
-          <li className="nav-item offset-md-1">
-            <div className="dropdown">
-
-              <a data-target="#" href="page.html"
-                data-toggle="dropdown" className="dropdown-toggle">
-                <b className="caret"></b>
-              </a>
-              <ul className="dropdown-menu">
-                <li className='logout'><a href="#"
-                onClick={this.handleLogout}>Logout</a></li>
 
-              </ul>
+          <li className="nav-item dropdown">
+            <a className="nav-link d-inline-flex text-capitalize" href="#" id="navbarDropdown"
+               role="button" data-toggle="dropdown" aria-haspopup="true"
+               aria-expanded="false"> {user.registeredUsername || user.username}<i className='fa fa-caret-down m-2'/>
+            </a>
+            <div className="dropdown-menu" aria-labelledby="navbarDropdown">
+              <NavLink to="/profile" className="dropdown-item">My Profile</NavLink>
+              <NavLink to="/create-art" className={`dropdown-item ${userType === 'artist' ? '' : 'd-none'}`}>Create Art</NavLink>
+              <div className="dropdown-divider"/>
+              <NavLink to="/login" onClick={this.handleLogout} id="nav-link" className="dropdown-item">logout</NavLink>
```

Comment:
Are you sure you want to log the user out when you click on the login link?

Reply:
I'll make the corrections. Thanks